### DISCUSSION TITLE
Can Link to panels with supported chips be provided

### DISCUSSION THREAD

**Discussion by janvandeklokHome** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#583] [General]
Hi,  You've done a fantastic job here!!!      There just 1 thing which would help a lot: A list with links to panels that have the supported chips.  

Almost no supplier lists the chips used on their boards..  So I thought if there are panels known to work with the software, you could collect links to the panels (to their suppliers) and publish them here. 

Keep up the good work!!!

Jan

**Comment by board707** [comment]
> It is impossible to provide a comprehensive list of what panels are supported (or not supported) ...

https://github.com/mrfaptastic/ESP32-HUB75-MatrixPanel-DMA?tab=readme-ov-file#supported-panel-can-types

**Comment by janvandeklokHome** [comment]
Ok Thanks.  I had read that section but the problem is that almost no supplier mentions the IC type of its panel. Thats why I thought it might be helpful  provide such a list.   F.I. this one https://nl.aliexpress.com/item/1005003218018477.html?spm=a2g0o.order_list.order_list_main.5.332c79d2bqMc32&gatewayAdapt=glo2nld
I have ordered a couple because the seller claims it contains the FM6126A AKA ICN2038S  IC.   I will let you know if they really do when they arrive....
Thanks.

**Reply by mrcodetastic** [reply]
You need to contact the suppliers directly and confirm before ordering,  and use that as evidence in the case they don't send what they said they would. I can't help here.
